### PR TITLE
Generate game pages to minimise copy&paste

### PR DESCRIPTION
**Pull request by marekventur**
To add new games and languages just add them to ```games.js```.

### CODE REVIEW

File: the-heist.html [hunk 1]
```diff
@@ -1,29 +1,7 @@
...
-</div>
-
-@@include('./partials/footer.html')
+<!doctype html>
+<html lang="en">
+<head>
+  <meta http-equiv="refresh" content="0; url=/games/the-heist.html">
+</head>
+<body></body>
+</html>
```
Comment (uorbe001) [inline the-heist.html]: can't we simply delete these and generate them from the partial?

File: games.js [hunk 2]
```diff
@@ -0,0 +1,74 @@
+var GAMES = [
+    {
+        "code": "the-heist",
```
Comment (uorbe001) [inline games.js]: can we call this path or something similar? 'code' sounds like the "gameCode" that we are using to identify the games in the server, and the naming is not consistent with this, so I think it'd be confusing

File: partials/footer.html [hunk 3]
```diff
@@ -37,7 +37,7 @@
     <!-- JavaScript
     ================================================== -->
 
-    <script src="main.js"></script>
+    <script src="/main.js"></script>
```
Comment (uorbe001) [inline partials/footer.html]: out of curiosity, why is this needed now?

File: the-heist.html (hunk 1 above)
Comment (marekventur) [inline the-heist.html]: Apparently we have given the urls out to too many people already, I'd be happy to throw them away otherwise.

File: games.js (hunk 2 above)
Comment (marekventur) [inline games.js]: :+1:

File: the-heist.html (hunk 1 above)
Comment (marekventur) [inline the-heist.html]: I've just had a much better idea. We don't need that /games/folder at all, let's get rid of it.

File: partials/footer.html (hunk 3 above)
Comment (marekventur) [inline partials/footer.html]: to allow pages like "gamevy.com/folder/something.html" to fetch the js from "gamey.com/main.js" instead of "gamevy.com/folder/main.js". I should have done it like this from the beginning.

Comment (uorbe001) [inline partials/footer.html]: ah, makes sense

**Comment by Nahikariii** [comment]
Nice!
